script/os-autoinst-setup-multi-machine

```diff
@@ -45,11 +45,13 @@ install_packages() {
 
 configure_firewall() {
     systemctl enable --now firewalld
+    default_zone=$(firewall-cmd --get-default-zone)
     firewall-cmd --permanent --get-services | grep -q isotovideo && firewall-cmd --permanent --delete-service=isotovideo
     firewall-cmd --permanent --new-service isotovideo
     for i in $(seq 1 "$instances"); do firewall-cmd --permanent --service=isotovideo --add-port=$((i * 10 + 20003))/tcp; done
     firewall-cmd --permanent --zone="$zone" --add-service=isotovideo
-    firewall-cmd --get-default-zone | grep -q "$zone" || firewall-cmd --set-default-zone="$zone"
+    [[ "$default_zone" == "$zone" ]] || firewall-cmd -q --permanent --remove-interface="$ethernet" --zone="$default_zone"
+    [[ "$default_zone" == "$zone" ]] || firewall-cmd --set-default-zone="$zone"
```

Comment:
```suggestion
   if  [[ $default_zone == $zone ]]; then
     firewall-cmd -q --permanent --remove-interface="$ethernet" --zone="$default_zone"
     firewall-cmd --set-default-zone="$zone"
   fi
```
No need for check twice the condition. wdyt?

Reply:
Nope, wrong condition here `||` is equivalent to `if not`